docs/contributing.md

```diff
@@ -0,0 +1,4 @@
+(contributing-guide) =
```

Comment:
This line `(contributing-guide) =` appears in the browser version, is it needed?